SPECS/libxcrypt/libxcrypt.spec

```diff
@@ -142,12 +142,7 @@ Provides:       %{name}-common = %{version}-%{release}
 Obsoletes:      %{name}-compat < %{version}-%{release}
 %endif
 # We need a version of glibc, that doesn't build libcrypt anymore.
-#Requires:       glibc%{?_isa}         >= %{glibc_minver}
-%if %{with override_glibc}
```

Comment:
Could you also remove these few lines from the spec, since they aren't used anymore:
```
# override_glibc and glibcversion are temporary to make libxcrypt install on top of glibc
%define glibcversion 2.38
%bcond_with override_glibc
```

Reply:
I only noticed this right now: since we've been using `%bcond_with override_glibc` in the spec, it looks like `%{with override_glibc}` sections would always be skipped. We don't set any `with X` macros, if I'm not mistaken. I'm fine doing the clean-up, if it's only specific to our spec, but I just wanted to point this out, in case we're expecting any functional changes after this PR is in.

Reply:
Yep will remove those lines.
All this override_glibc logic was added by us to workaround an issue with adding this improved version of `libcrypt` into 1.0 while it was already released. And in 1.0, 2.0, and pre-Nov 2023 3.0, the line was `%bcond_without override_glibc`, but you're right that today `%bcond_with override_glibc` means all these overrides are inert and are diffs from the original spec.

The only real functional change is the requires glibc that you highlighted.